.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
```

Comment:
- [ ] Please make sure that the boarder of the card have a color that match the design 
- [ ] Please make sure to have the same font family as in the design for the card title and the tech list 
Your disgn 
![image](https://user-images.githubusercontent.com/74744787/212491737-0d099efa-9c26-451d-84c2-97ab3d1272a8.png)
Figma 
![image](https://user-images.githubusercontent.com/74744787/212491750-77e47938-bf7a-4a14-94de-4fc3d3f2a609.png)